client/finality-grandpa/src/communication/mod.rs

```diff
@@ -97,50 +95,6 @@ mod benefit {
 	pub(super) const PER_EQUIVOCATION: i32 = 10;
 }
 
-/// A handle to the network. This is generally implemented by providing some
-/// handle to a gossip service or similar.
-///
-/// Intended to be a lightweight handle such as an `Arc`.
-pub trait Network<Block: BlockT>: Clone + Send + 'static {
```

Comment:
I wouldn't mind something like this going into `consensus-common` either - where we can focus on the properties a network needs to provide to a consensus engine.